packages/components/package.json

```diff
@@ -40,7 +38,8 @@
     "release": "semantic-release  --tagFormat 'components/${version}' -e semantic-release-monorepo",
     "release.angular": "git add ../angular && git commit -m \"chore(release/angular) via component release [skip actions]\"",
     "release.react": "git add ../react && git commit -m \"chore(release/react) via component release [skip actions]\"",
-    "release.vue": "git add ../vue && git commit -m \"chore(release/vue) via component release [skip actions]\""
+    "release.vue": "git add ../vue && git commit -m \"chore(release/vue) via component release [skip actions]\"",
+    "release.dry": "semantic-release --dry-run --tagFormat 'components/${version}' -e semantic-release-monorepo"
```

Comment:
Do you think it would make sense to add this as sort of a "semi verify-step" to `CONTRIBUTION.md`?

Reply:
Locally it breaks because of missing GitHub tokens, but on GitHub we could implement it into our Quality Gate. Will do that directly.